tests/kickstart_tests/user.ks

```diff
@@ -0,0 +1,80 @@
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+
+install
+
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+
+clearpart --all
+autopart --type=lvm
```

Comment:
This should just be autopart since we're not testing autopart type (and to be consistent with the other kickstarts)